Makefile

```diff
@@ -0,0 +1,59 @@
+##@ Makefile
+
+##@ Define the default version
+VERSION ?= latest
+
```

Comment:
Can we declare the URL parts as const to easily install a pre-release version?
EG:
ROOT_URL ?= `https://connectors.airbyte.com/airbyte-ci/releases`
VERSION  ?= `latest`

* `ROOT_URL` could be tweaked to download from `https://storage.googleapis.com/dev-airbyte-cloud-connector-metadata-service`
* `VERSION` could be tweaked to point a specific commit.

Let's say I want to use the binary from this branch:
```
ROOT_URL = `https://storage.googleapis.com/dev-airbyte-cloud-connector-metadata-service
VERSION = <head-commit-id-from-this-branch>
```

Reply:
Yup! Love this

Reply:
I think you did not expose the `ROOT_URL`var

Reply:
I didnt expose it as an argument. I opted to leave it as an environment variable just so if you needed to override it you could do so from anywhere where. Even if the script was called by another script

<img width="1351" alt="image" src="https://github.com/airbytehq/airbyte/assets/1325608/9187d394-4237-40ed-b410-95c8b6a573c4">

Reply:
👍